be/src/olap/column_predicate.h

```diff
@@ -91,6 +91,9 @@ class ColumnPredicate {
 protected:
     uint32_t _column_id;
     bool _opposite;
+    mutable uint64_t _evaluated_rows = 1;
```

Comment:
the value only use in bloom filter predicate. it should not define here?

Reply:
Yes. My mistake.